mtp_api/apps/prison/tests/test_views.py

```diff
@@ -567,6 +568,38 @@ def test_retrieving_combined_balance_in_private_estate(self, mocked_nomis):
         self.assertDictEqual(response_data, {'combined_account_balance': 0})
         mocked_nomis.get_account_balances.assert_not_called()
 
+    @mock.patch('prison.serializers.nomis')
```

Comment:
Might be nicer to use `RequestsMock` here to standardise how we mock external services across the coddebase(s), but i'm not going to block over this